contrib/epee/include/net/http_server_handlers_map2.h

```diff
@@ -152,6 +152,16 @@
        epee::serialization::store_t_to_json(static_cast<epee::json_rpc::error_response&>(rsp), response_info.m_body); \
        return true; \
     } \
+    epee::serialization::storage_entry se; \
+    if(ps.get_value("params", se, nullptr) && se.type() != typeid(epee::serialization::section)) \
+    { \
+       boost::value_initialized<epee::json_rpc::error_response> rsp; \
```

Comment:
`boost::value_initialized` isn't really needed anymore with C++11 brace initialization `{}`. Although it doesn't hurt, it's just wasted include times at this point. And changing to that style means all of the `static_cast`s below can be removed too.

Reply:
I'll leave it as is, that's what the surrounding code does and it is not buggy.